src/specs/networking.md

```diff
@@ -66,3 +68,26 @@ Accepting a `WireTxPayForMessage` into the mempool requires different logic than
+|---------------|---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `shareProofs` | [ShareProof](./data_structures.md#shareproof)`[]`                                           | The available shares in the offending row or column.                                                                |
+| `root`        | [HashDigest](./data_structures.md#hashdigest)                                               | The Merkle root of the offending row or column.                                                                      |
+| `proof`       | [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) | The Merkle proof of the row or column root in [`availableDataRoot`](./data_structures.md#header).|
+| `isCol`       | `bool`                                                                                      | A Boolean indicating if it is an offending row or column; `false` if it is a row.                                    |
+| `position`    | `uint64`                                                                                    | The index of the row or column in the square.                                                                        |
+
+
+## StateFraudProof
+
+| name                       | type                                                                                     | description                                                                                                                                                                                                                                                                                  |
+|----------------------------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `headerForTxns`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to transaction data.                                                                                              |
+| `headerForISRs`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to intermediate state root data. `lastHeaderHash` within `headerForISRs` equals the hash of `headerForTxns`.|
```

Comment:
I'm not sure we need these two fields. The verifier should know the headers for all blocks I think. This is not actually obviously the case. We could have a model where the verifier only has the hash of the latest header and nothing else (not even the validator set or all previous header hashes). We should decide on a verifier model before committing to a particular fraud proofs scheme here.

Reply:
Headers are removed from the data structure for StateFraudProof for the time being.